### PR TITLE
Popularity trend graph is complete

### PR DESCRIPTION
**Pull request by relhajj**
While reviewing the popularity trend graph I made before vacation, I made some test cases for all extreme conditions I could think of and ended up catching a few errors that are now all fixed! So now we have a popularity trend line graph for each domain. Each graph has the maximal capacity of two years worth of data points (one point per month). In order to allow for the x-axis label text to be seen by viewers and not overlap, I had to make the decision that only the first data point and first month of each year include the year (2019,2020 etc.). Otherwise, only the month is included as the x-axis label (1,2,3...). As well, to make sure that only 24 points are shown in the graph at most, I hard coded the start date of when we started to collect data (june 2019). This could be changed but I had to make this decision to test out the data and see how it looks. I will send you pictures I took of the graphs after the code is run on slack.

### CODE REVIEW

**Comment by snadi** [comment]
Instead of hard-coding the start date, can't we just go two years back from today? If there is less data than two years, then we have it all.

File: scripts/filldb.py [hunk 1]
```diff
@@ -459,14 +503,20 @@ def createCharts():
...
+  fillPopularityData(entrymonth,entryyear)
+  createCharts(entrymonth,entryyear)
+  fillReleaseFrequencyData()
+  fillBreakingChanges()
+  fillLastModificationDateData()
+  fillLastDiscussedSOData()
+  fillLicenseData()
+  fillIssueClosingTimeData()
+  fillIssueResponseTimeData()
+  fillIssueData()
```
Comment (snadi) [inline scripts/filldb.py]: are these indented correctly?

**Comment by snadi** [comment]
Will there need to be corresponding changes in the website to show the popularity trend graph?

File: scripts/filldb.py (hunk 1 above)
Comment (relhajj) [reply scripts/filldb.py]: Yes they should be, the code ran fine when I ran it

**Comment by snadi** [comment]
So everything to do with fill the DB or creating charts is in this repo in filldb.py

**Pull request merged**